src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,36 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name) {
+    super(health, name);
```

Comment:
The constructor of the Herbivore class should not accept a 'health' parameter as the health is always set to 100 by default according to the task description. You should only pass the 'name' parameter to the super constructor.